library/built-in/_prefabs/prefab_latent_v3.ts

```diff
@@ -3,19 +3,32 @@ import type { OutputFor } from './_prefabs'
 
 export const ui_latent_v3 = () => {
     const form: FormBuilder = getCurrentForm()
+    const batchSize = form.int({ step: 1, default: 1, min: 1, max: 8 })
```

Comment:
This I need to fix the behaviour and/or find the right API before we can merge this PR.🙃